proxy.rst

```diff
@@ -29,3 +29,5 @@ If you're going to run something using sudo, remember to use the ``-i`` option t
 ::
 
     sudo -i so-rule-update
+
+It won't work if you do a ``sudo su -`` followed by a ``so-rule-update``.
```

Comment:
`sudo su -` ignores `/etc/environment`, but `sudo su` does not.
 
I would probably reword this as:
```rst
.. warning::
	Using ``sudo su -`` will ignore ``/etc/environment``, instead use ``sudo su`` if you need to operate as root.
```